add-ons/fabric8-launcher/fabric8-launcher.addon

```diff
@@ -0,0 +1,20 @@
+###################################################################################################
+# Name: fabric8-launcher
+# Description: Installs Fabric8 Launcher in a project named rhoarpad 
+# Required-Vars: GITHUB_USERNAME, GITHUB_TOKEN 
+# OpenShift-Version: >3.9.0       
+# Var-Defaults: OPENSHIFT_USERNAME=developer,CONSOLE_URL=https://#{ip}:8443,LAUNCHER_PROJ=rhoarpad
+# Url: Template from https://raw.githubusercontent.com/fabric8-launcher/launcher-openshift-templates/master/openshift/launcher-template.yaml
+###################################################################################################
+
+# login
+oc login -u #{OPENSHIFT_USERNAME} -p ' '
+
+# create project
+oc new-project #{LAUNCHER_PROJ} 
+
+# add template
+oc create -n #{LAUNCHER_PROJ} -f https://raw.githubusercontent.com/fabric8-launcher/launcher-openshift-templates/master/openshift/launcher-template.yaml
+
+# instantiate launcher
+oc new-app fabric8-launcher --name=launcher -n #{LAUNCHER_PROJ} -p LAUNCHER_MISSIONCONTROL_GITHUB_USERNAME=#{GITHUB_USERNAME} -p LAUNCHER_MISSIONCONTROL_GITHUB_TOKEN=#{GITHUB_TOKEN} -p LAUNCHER_MISSIONCONTROL_OPENSHIFT_CONSOLE_URL=#{CONSOLE_URL} -p LAUNCHER_MISSIONCONTROL_OPENSHIFT_API_URL=#{CONSOLE_URL} -p LAUNCHER_MISSIONCONTROL_OPENSHIFT_USERNAME=#{OPENSHIFT_USERNAME} -p LAUNCHER_MISSIONCONTROL_OPENSHIFT_PASSWORD=#{OPENSHIFT_PASSWORD}  
```

Comment:
@VeerMuchandi Here you put `#{OPENSHIFT_PASSWORD}` which is not defined as Required-Vars or in the Var-Defaults, what I suggest to have is something below. So that username and password also can be overwritten if needed.

```
###################################################################################################
# Name: fabric8-launcher
# Description: Installs Fabric8 Launcher in a project named rhoarpad 
# Required-Vars: GITHUB_USERNAME, GITHUB_TOKEN , OPENSHIFT_USERNAME, OPENSHIFT_PASSWORD
# OpenShift-Version: >3.9.0       
# Var-Defaults: OPENSHIFT_USERNAME=developer,CONSOLE_URL=https://#{ip}:8443,LAUNCHER_PROJ=rhoarpad, OPENSHIFT_PASSWORD=developer
# Url: Template from https://raw.githubusercontent.com/fabric8-launcher/launcher-openshift-templates/master/openshift/launcher-template.yaml
###################################################################################################

# login
oc login -u #{OPENSHIFT_USERNAME} -p #{OPENSHIFT_PASSWORD}

# create project
oc new-project #{LAUNCHER_PROJ} 

# add template
oc create -n #{LAUNCHER_PROJ} -f https://raw.githubusercontent.com/fabric8-launcher/launcher-openshift-templates/master/openshift/launcher-template.yaml

# instantiate launcher
oc new-app fabric8-launcher --name=launcher -n #{LAUNCHER_PROJ} -p LAUNCHER_MISSIONCONTROL_GITHUB_USERNAME=#{GITHUB_USERNAME} -p LAUNCHER_MISSIONCONTROL_GITHUB_TOKEN=#{GITHUB_TOKEN} -p LAUNCHER_MISSIONCONTROL_OPENSHIFT_CONSOLE_URL=#{CONSOLE_URL} -p LAUNCHER_MISSIONCONTROL_OPENSHIFT_API_URL=#{CONSOLE_URL} -p LAUNCHER_MISSIONCONTROL_OPENSHIFT_USERNAME=#{OPENSHIFT_USERNAME} -p LAUNCHER_MISSIONCONTROL_OPENSHIFT_PASSWORD=#{OPENSHIFT_PASSWORD} 
```